src/pages/strategy/tabs/ships/ships.js

```diff
@@ -238,6 +238,54 @@
 			$(".pages_no").on("click", function(){
 				$(".ingame_page").hide();
 			});
+            
+            /* Filter for duplicates */
+            $(".dupe_filter").on("click", function() {
+                if (!$(this).hasClass("active")) {
+                    $(".ship_list .ship_item").each(function() {
+                        var thisShipId = parseInt($(this).find(".ship_id").text());
+                        /* Find master ID */
+                        var result = $.grep(KC3StrategyTabs.ships.definition.shipCache, function(e) {
+                            return e.id === thisShipId;
+                        });
+                        var dupeCount = 0;
+                        function getDupeShipsCount() {
+                            var dupeCheck = $.grep(KC3StrategyTabs.ships.definition.shipCache, function(e) {

```

Comment:
this part is really strange... `getDupeShipsCount` return nothing but set the `dupeCount `, declared unused `dupeCheck` var. if condition of dupe: true is matching the master id of base remodel, might use `RemodelDb.originOf` on two sides of `bid` (master id of ship)